channel.go

```diff
@@ -27,6 +27,8 @@ type ChannelMember struct {
 	InviteAcceptedAt *time.Time `json:"invite_accepted_at,omitempty"`
 	InviteRejectedAt *time.Time `json:"invite_rejected_at,omitempty"`
 	Role             string     `json:"role,omitempty"`
+	ArchivedAt       *time.Time `json:"archived_at,omitempty"`
```

Comment:
```suggestion
```

Reply:
it's still work in progress, i'll convert it to draft